pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -921,6 +937,117 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
         }
     }
 
+    // Circuit Breakers
+
+    /**
+     * @notice Return the circuit breaker parameters for the given token.
+     * @dev These are the reference values (BPT price and weight complement) computed when the breaker was set,
+     * along with the percentage bounds. To get the current BPT price boundaries needed to check whether the
+     * circuit breaker should trip, call `getCurrentCircuitBreakerBounds`.
+     */
+    function getCircuitBreakerFields(IERC20 token)
```

Comment:
I'd merge this with `getCurrentBounds` and even `getBptPrice`, and return a single thing with the entire state. Realistically they'll all be used at the same time, and they all read essentially the same data.